src/main/java/seedu/address/logic/commands/ConfirmationRejectCommand.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+
+/**
+ * Rejects the execution of a Dangerous Command.
```

Comment:
Perhaps the space between "Dangerous" and "Command" can be removed?

Reply:
Resolved!